examples/memeolist.tables.sql

```diff
@@ -1,6 +1,21 @@
-DROP TABLE IF EXISTS "Meme";
+CREATE TABLE Profile (
```

Comment:
Can we add back in the following here:

```
DROP TABLE IF EXISTS "Meme";
DROP TABLE IF EXISTS "Profile";
DROP TABLE IF EXISTS "Comment";
```

It just makes local dev a lot simpler